src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -2218,6 +2213,9 @@ public bool MapCompileTimeSrcToCurrentSrc(string compilerSrc, out string current
         {
             if (_launchOptions.SourceMap != null)
             {
+                // Convert to Client source paths
+                compilerSrc = PlatformUtilities.PathToHostOSPath(compilerSrc);
```

Comment:
>compilerSrc [](start = 16, length = 11)

This is going to change the return value even in the case that we don't find the path (see line 2249), but only if there is a source map. You probably instead want to assign to a new local, and change the uses of `compilerSrc` on line 2219 to 2247 to use your new local variable.